datumaro/plugins/mpii_format.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os.path as osp
+
+import scipy.io as spio
+
+from datumaro.components.annotation import (
+    Bbox, LabelCategories, Points, PointsCategories,
+)
+from datumaro.components.extractor import (
+    AnnotationType, DatasetItem, Importer, SourceExtractor,
+)
+from datumaro.components.format_detection import FormatDetectionContext
+from datumaro.components.media import Image
+
+MPII_POINTS_LABELS = [
+    'r_ankle',
+    'r_knee',
+    'r_hip',
+    'l_hip',
+    'l_knee',
+    'l_ankle',
+    'pelvis',
+    'thorax',
+    'upper_neck',
+    'head top',
+    'r_wrist',
+    'r_elbow',
+    'r_shoulder',
+    'l_shoulder',
+    'l_elbow',
+    'l_wrist'
+]
+
+MPI_POINTS_JOINTS = [
+    (0, 1), (1, 2), (2, 6), (3, 4),
+    (3, 6), (4, 5), (6, 7), (7, 8),
+    (8, 9), (8, 12), (8, 13), (10, 11),
+    (11, 12), (13, 14), (14, 15)
+]
+
+class MpiiExtractor(SourceExtractor):
+    def __init__(self, path):
+        if not osp.isfile(path):
+            raise FileNotFoundError("Can't read annotation file '%s'" % path)
+
+        super().__init__()
+
+        self._categories = {
+            AnnotationType.label: LabelCategories.from_iterable(['human']),
+            AnnotationType.points: PointsCategories.from_iterable(
+                [(0, MPII_POINTS_LABELS, MPI_POINTS_JOINTS)])
+        }
+
+        self._items = list(self._load_items(path).values())
+
+    def _load_items(self, path):
+        items = {}
+
+        root_dir = osp.dirname(path)
+
+        data = spio.loadmat(path, struct_as_record=False, squeeze_me=True).get('RELEASE', {})
+        data = data.__dict__['annolist']
```

Comment:
Why are you using `__dict__` here? It should allow to access members just with a dot like `obj.member`. If you need to check if a member exists, use `hasattr` and `getattr`.